av1-rtp-spec.md

```diff
@@ -216,12 +223,23 @@ The design allows combination of aggregation and fragmentation, i.e., allow for
 a set of OBUs in which the first and/or last one is fragmented.
 
 The payload contains a series of one or more OBUs (with the first and/or last
-possibly being fragments). Each OBU (or OBU fragment) is preceded by a length
-field. The length field is encoded using leb128. Leb128 is defined in the AV1
+possibly being fragments).
+
+The length field is encoded using leb128. Leb128 is defined in the AV1
 specification, and provides for a variable-sized, byte-oriented encoding of non-
 negative integers where the first bit of each (little-endian) byte indicates if
 additional bytes are used in the representation (AV1, Section 4.10.5).
 
+Whether or not the first and/or last OBU is fragmented is signaled in the
+aggregation header. Fragmentation may occur regardless of how the X flag
+is set.
+
+The AV1 specification allows OBUs to have an optional size field called 
+obu_size (also leb128 encoded), signaled by the obu_has_size_field flag 
+in the OBU header. To minimize overhead, the obu_has_size_field flag SHOULD 
+be set to zero in all OBUs except fragmented OBUs where it MUST be set to

```

Comment:
Why signal the OBU size (in the bitstream) for fragmented OBUs? The RTP payload format lets us infer the size of each fragment, and the Z,Y flags helps us find the first and last fragment. The parser should be able to assemble all  fragments and just sum their sizes.

Reply:
Fixed.